### PR TITLE
 Batch library versioning into versions.gradle 

### PR DESCRIPTION
**Pull request by mTwTm**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,21 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    apply from: 'versions.gradle'
+    addRepos(repositories)
```
Comment (benjamin-cheng) [inline build.gradle]: Following `repositories {...}` is replaced by `addRepos(repositories)` so should be no longer needed.

File: build.gradle [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+        classpath deps.dicedmelon_jacoco_plugin
+        classpath deps.gms_plugin
+        classpath deps.kotlin.plugin
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
 allprojects {
+    addRepos(repositories)
```
Comment (benjamin-cheng) [inline build.gradle]: Following `repositories {...}` is replaced by `addRepos(repositories)` so should be no longer needed.

File: app/build.gradle [hunk 3]
```diff
@@ -23,15 +23,15 @@ buildscript {
...
     defaultConfig {
         applicationId "org.mozilla"
-        minSdkVersion Versions.min_sdk
-        targetSdkVersion Versions.target_sdk
+        minSdkVersion build_versions.min_sdk
+        targetSdkVersion build_versions.target_sdk
         resConfigs "en", "in"
-        versionCode Versions.version_code
-        versionName Versions.version_name
+        versionCode 1
```
Comment (benjamin-cheng) [inline app/build.gradle]: Why not also put both `versionCode` and `versionName` settings into `versions.gradle`?

**Comment by mTwTm** [comment]
Ref: https://android-developers.googleblog.com/2018/05/announcing-new-sdk-versioning.html?m=1

**Pull request closed without merging**